src/bitmessagemain.py

```diff
@@ -338,19 +287,9 @@ def start(self):
             shared.reloadBroadcastSendersForWhichImWatching()
 
             # API is also objproc dependent
-            if BMConfigParser().safeGetBoolean(
-                    'bitmessagesettings', 'apienabled'):
-                try:
-                    apiNotifyPath = BMConfigParser().get(
-                        'bitmessagesettings', 'apinotifypath')
-                except:
-                    apiNotifyPath = ''
-                if apiNotifyPath != '':
-                    with shared.printLock:
-                        print('Trying to call', apiNotifyPath)
-
-                    call([apiNotifyPath, "startingUp"])
-                singleAPIThread = singleAPI()
+            if BMConfigParser().safeGetBoolean('bitmessagesettings', 'apienabled'):
+                import api
```

Comment:
Just add a disable comment for pylint, then the check will succeed and you can merge (I already approved).